src/app/chat/views/message_format.nim

```diff
@@ -31,14 +31,19 @@ proc renderInline(self: ChatMessageList, elem: TextItem): string =
   else: result = fmt(" {value} ")
 
 # See render-block in status-react/src/status_im/ui/screens/chat/message/message.cljs
-proc renderBlock(self: ChatMessageList, message: Message): string =
+proc renderBlock(self: ChatMessageList, message: Message, isEdited: bool = false): string =
+  var i = 0
   for pMsg in message.parsedText:
+    i = i + 1
     case pMsg.textType:
       of "paragraph": 
         result = result & "<p>"
         for children in pMsg.children:
           result = result & self.renderInline(children)
-        result = result & "</p>"
+        if i == message.parsedText.len and isEdited:
+          result = result & "__EDITLABEL__" & "</p>"
```

Comment:
I originally tried, but I could not make it match the design, since in the design the "Edited" label follows the text and my implementation ended up in the wrong position. Another option was to use js to parse the content of the message and add the label manually to the dom of the message but that did not look efficient